dbt/adapters/spark/session.py

```diff
@@ -9,6 +9,7 @@
 from dbt.events import AdapterLogger
 from dbt.utils import DECIMALS
 from pyspark.sql import DataFrame, Row, SparkSession
+from dbt.spark.adapters import ConnectionWrapper
```

Comment:
Wouldn't this line and it's reference below need to be updated? I'm surprised this passed tests; normally we'd import from `dbt.adapters.spark`, and in this case `dbt.adapters.spark.connections` since this class isn't in `dbt.adapters.spark.__init__.py`.

Reply:
Probably because we have the spark session tests disabled

Reply:
Hmm, not sure how this happened, and also a bit scary that the linter didn't catch this.